pzp/processing/danger_zones.py

```diff
@@ -61,6 +63,17 @@ def initAlgorithm(self, config=None):
             )
         )
 
+        self.addParameter(
+            QgsProcessingParameterField(
```

Comment:
Is the value coming from a vector layer field or could we use [QgsProcessingParameterNumber](https://api.qgis.org/api/master/classQgsProcessingParameterNumber.html) instead?